.pylintrc

```diff
@@ -319,7 +319,7 @@ ignored-classes=optparse.Values,thread._local,_thread._local
 # (useful for modules/projects where namespaces are manipulated during runtime
 # and thus existing member attributes cannot be deduced by static analysis). It
 # supports qualified module names, as well as Unix pattern matching.
-ignored-modules=matplotlib
+ignored-modules=matplotlib,sympy,pandas,numpy
```

Comment:
what was the problem?

Reply:
pylint complained about correct imports from there.